### ISSUE TITLE
User can change pathway target and event cohorts names in read-only mode

### ISSUE THREAD

**Issue by ttemnikova** [OHDSI/Atlas#2716]
### Expected behavior
A read-only cohort pathway object does not look like it may be changed

### Actual behavior
A user can change the target and event cohort names (but cannot save the changes)
![image](https://user-images.githubusercontent.com/95474802/175081701-387a9fc2-d7fa-48f5-82a4-0a9f112ca146.png)

### Steps to reproduce behavior
1. Login as userA
2. Create a cohort pathway
3. Add a target cohort and an event cohort
4. Save
5. Login as userB
6. Open the cohort created
7. Change the name of the target and event cohort

**Comment by chrisknoll** [comment]
Fair point, we should remove the styling on the cohort name if it is read only.
